### PR TITLE
[Lang] Fix pylint rule W0235

### PR DESCRIPTION
**Pull request by tssujt**
Related issue = #3151

### CODE REVIEW

File: python/taichi/ui/scene.py [hunk 1]
```diff
@@ -134,8 +131,5 @@ def particles(self,
         vbo_info = get_field_info(vbo)
         super().particles(vbo_info, has_per_vertex_color, color, radius)
 
-    def point_light(self, pos, color):
+    def point_light(self, pos, color):  # pylint: disable=W0235
```
Comment (k-ye) [inline python/taichi/ui/scene.py:134]: Why handling this differently :-) ?

**Review by k-ye — APPROVED** [review]
Thanks, welcome to the community!

File: python/taichi/ui/scene.py (hunk 1 above)
Comment (tssujt) [reply python/taichi/ui/scene.py:134]: https://github.com/taichi-dev/taichi/runs/4191773093

@k-ye CI fails, but I'm not familiar with pybind11.

**Comment by tssujt** [comment]
Thanks @sjwsl

**Pull request merged**